src/styles/navbar.css

```diff
@@ -0,0 +1,139 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600&display=swap');
+/* My Component Library link  */
+@import url("https://rebalance-ui.netlify.app/all_component.css");
+
+.home {
+    background-color: var(--bg-color);
+    padding: 13rem 6%;
+}
+
+section {
+    padding: 6rem 6%;
+}
+
+.heading {
+    text-align: center;
+    margin-bottom: 2rem;
+    position: relative;
+}
+
+.heading::before {
+    content: '';
+    position: absolute;
+    top: 50%;
+    left: 0;
+    transform: translateY(-50%);
+    width: 100%;
+    height: .01rem;
+    background: rgba(0, 0, 0, .1);
+    z-index: -1;
+}
+
+.heading span {
+    font-size: 3rem;
+    padding: .5rem 2rem;
+    color: var(--black);
+    background: var(--white-hover);
+    border: 2px solid var(--secondary-gray);
+    border-radius: 6px;
+    cursor: pointer;
+}
+
+.heading span:hover {
+    border-color: var(--yellow-color);
+}
+
+.btn {
+    margin-top: 1rem;
+    display: inline-block;
+    padding: .9rem 3rem;
+    border-radius: .5rem;
+    color: black;
+    background: var(--yellow-color);
+    font-size: 1.7rem;
+    cursor: pointer;
+    font-weight: 500;
+}
+
+.btn:hover {
+    border: 2px solid var(--white-hover);
+}
+
+.header-bookhub {
+    position: fixed;
+    left: 0;
+    top: 0;
+    right: 0;
+    background-color: var(--bg-color);
+    padding: 1.5rem 4%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    z-index: 999;
```

Comment:
On my last PR review, someone told me not to give this much z-index value. You can change it if you want rest looks good to me.

Reply:
okay...done